src/tools/device.cpp

```diff
@@ -527,7 +523,6 @@ void Device::integrate( const double & dt )
   {
     assert(state_.size()==controlIN.size()+6);
     state_.tail(controlIN.size()) = controlIN;
-    CHECK_BOUNDS(state_, lowerPosition_, upperPosition_, "position");
```

Comment:
What is the reason for removing this ?

Reply:
When no integration is done, the state_ is not making sense and specifically when using torque control. The motor control is current and this means that we should have an actuator model to make a correct integration. This is not yet available.